webapp/src/components/table/tableRow.tsx

```diff
@@ -140,6 +147,28 @@ const TableRow = React.memo((props: Props) => {
                             />
                         </div>)
                 })}
+
+            <MenuWrapper
+                className='optionsMenu'
+                stopPropagationOnToggle={true}
+            >
+                <IconButton icon={<OptionsIcon/>}/>
+                <Menu position='left'>
+                    <Menu.Text
+                        icon={<DeleteIcon/>}
+                        id='delete'
+                        name={'Delete'}
```

Comment:
Should internalization be used for this `name`?

Reply:
Hi @kamre , so I need to  add "TableRow.delete" in internalization file, right ?

Reply:
@arvinDarmawan yes, there are many other usages of `*.delete` as `id` for formated messages in other places.

So it looks like `TableRow.delete` is the correct name for `id` in this place.

Reply:
Hi @kamre , I already check again in en.json file, and there are  "ContentBlock.Delete" and "TableHeaderMenu.delete".
Should we use delete with initial letter d or D ?

Reply:
@arvinDarmawan I don't have exact anwser, but it looks like most of other ids use lower case after the dot.

Reply:
done @kamre , just waiting for UX/design